views/artist-search-results.hbs

```diff
@@ -0,0 +1,4 @@
+{{#each artists}}
+{{this.name}}
+<img src="{{this.images.0.url}}" alt="">
+{{/each}}
```

Comment:
Here you could also add a link to the albums of the artist.

`<a href="/albums/{{this.id}}">See the albums</a>`  Here `this.id` refers to the artist id. You have created a route for this part in line 47 in app.js.